python/ray/monitor.py

```diff
@@ -575,9 +588,6 @@ def run(self):
     redis_ip_address = get_ip_address(args.redis_address)
     redis_port = get_port(args.redis_address)
 
-    # Initialize the global state.
-    ray.global_state._initialize_global_state(redis_ip_address, redis_port)
```

Comment:
Is this since we don't need to get the client table any more?

Reply:
I'm not 100% sure, but I think that was always redundant because the monitor creates its own global state at

https://github.com/ray-project/ray/blob/51fdbe3867d7e7e56d37fce1aad9fc49d87f1afa/python/ray/monitor.py#L85-L86